tests/api_fastapi/core_api/routes/public/test_task_instances.py

```diff
@@ -973,8 +978,8 @@ def test_return_TI_only_from_readable_dags(self, test_client, session):
             )
         response = test_client.get("/public/dags/~/dagRuns/~/taskInstances")
         assert response.status_code == 200
-        assert response.json["total_entries"] == 3
-        assert len(response.json["task_instances"]) == 3
+        assert response.json()["total_entries"] == 3
+        assert len(response.json()["task_instances"]) == 3
```

Comment:
I was wondering how this could pass. Just realized it's xfailed, thanks.

Reply:
Yes there's this: `@pytest.mark.xfail(reason="permissions not implemented yet.")`